form-validation.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable no-unused-vars */
+// strating with declaring variables to get form elements
+const form = document.getElementById('form');
+const personName = document.getElementById('personname');
+const email = document.getElementById('email');
+const textArea = document.getElementById('textarea');
+
+// defining a function for form input validation checks
+function validationCheck() {
+  // declaring variables to get form elements values
+  const personNameValue = personName.value;
+  const emailValue = email.value;
+  const textAreaValue = textArea.value;
+
+  // defining function to check input type errors
+  function checkError(i, message) {
+    const subForm = i.parentElement;
+    const small = subForm.querySelector('small');
+    subForm.className = 'sub-form invalid';
+    small.innerText = message;
+  }
+  // defining function if there's no error
```

Comment:
* Nice work but I noticed a bug when I enter my email address in capital letters no errors came up below is , Kindly fix it

https://github.com/Kaiserabbas/project-portfolio/assets/52497246/1c9c3e73-803d-4176-a0c3-7bcb62012a86